### ISSUE TITLE
Testing EmailUtils and making Emails-Related error codes

### ISSUE THREAD

**Issue by crchong1** [keepid/keepid_server#10]
I would start with testing if the href targets actually exist on the emails on all the functions in Emailutils. In addition, think about what kinds of email-related error codes could occur (including what happens if the file is missing? rn we are just returning null, but that is not very descriptive). I would take a look at User.UserMessage enums to try to get some ideas, and to copy the toJSON() method. 
I would make a new folder in the tests folder and call it Security and make a new file called EmailUtilsUnitTests. You can look at ValidationUtilsUnitTests.java for reference.  

This should take probably one or two days I think, and in the meantime I will look for other pieces of the code you can work on. 
One huge thing is thinking about how to build a dashboard, and how we can query data on the backend. You should try to skim this documentation here. When you are reading, you should select the Java (sync) driver tab at the top of the sub articles. 
https://docs.mongodb.com/manual/crud/ 

![image](https://user-images.githubusercontent.com/13875633/88221293-4669ee80-cc32-11ea-82b3-cffd389c41e6.png)

Basically, the dashboard will probably be querying our mongo database, so knowledge of queries will be useful.
